src/mavm.rs

```diff
@@ -110,6 +110,36 @@ impl From<Instruction<AVMOpcode>> for Instruction {
     }
 }
 
+/// The default instruction generating macro sometimes overridden by other functions for simplicity.
+/// This macro is exported globally.
+macro_rules! create {
```

Comment:
We can change the name to something else, but because this will be our most used macro by far, I'd prefer we keep it something short. Maybe some word like `opcode`. It's used hundreds of times in this and the optimizer PR, and given the first arg is always something like `Tget` we probably don't need something with an underscore. What do you think?

Reply:
I just committed a change to `opcode!` for now. We can always change it again if need be